SmartDeviceLink/SDLGlobals.h

```diff
@@ -43,6 +43,8 @@ extern void *const SDLConcurrentQueueName;
 - (void)setDynamicMTUSize:(NSUInteger)maxMTUSize forServiceType:(SDLServiceType)serviceType;
 - (NSUInteger)mtuSizeForServiceType:(SDLServiceType)serviceType;
 
++ (void)runSyncOnSerialSubQueue:(dispatch_queue_t)queue block:(void (^)(void))block;
+
 @end
```

Comment:
Should there also be a `runAsyncOnSerialSubQueue:block:`? method?

Reply:
It's not necessary because calling async from the serial sub-queue itself won't cause a crash. So you can just `dispatch_async` without potential bugs.